File: src/lib/ocl.py
import platform
import sys

osname = ''
if sys.platform == "linux" or sys.platform == "linux2":
    osname = 'linux'
elif sys.platform == "darwin":
    osname = 'darwin'
elif sys.platform == "win32":
    osname = 'win32'
elif sys.platform == "win64":
    osname = 'win64'

major_version = sys.version_info[0]
minor_version = sys.version_info[1]

__import__('ocl.%s%s.%s' % (major_version, minor_version, osname))

File: src/npmpackage/lib/napi/ocl.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var platform = 'linux';
if (process.platform == 'win32') {
    if (process.arch === 'x64') {
        platform = 'win64';
    }
    else {
        platform = 'win32';
    }
}
else if (process.platform == 'darwin') {
    platform = 'darwin';
}
var buildType = 'Release';
if (process.env.DEBUG) {
    buildType = 'Debug';
}
var oclLib = require(__dirname + '/../../build/' + buildType + '/opencamlib.' + platform + '.node');
exports.default = oclLib;
